modules/consentManagementGpp.js

```diff
@@ -29,28 +30,30 @@ const cmpCallMap = {
   'static': lookupStaticConsentData
 };
 
-/**
- * This function checks the state of the IAB gppData's applicableSection field (to ensure it's populated and has a valid value).
- * section === 0 represents a CMP's default value when CMP is loading, it shoud not be used a real user's section.
- *
- * TODO --- The initial version of the GPP CMP API spec used this naming convention, but it was later changed as an update to the spec.
- * CMPs should adjust their logic to use the new format (applicableSecctions), but that may not be the case with the initial release.
- * Added support just in case for this transition period, can likely be removed at a later date...
- * @param gppData represents the IAB gppData object
- * @returns true|false
- */
-function checkApplicableSectionIsReady(gppData) {
-  return gppData && Array.isArray(gppData.applicableSection) && gppData.applicableSection.length > 0 && gppData.applicableSection[0] !== 0;
+const GPP_SECTIONS = {
+  // https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/main/Sections/Section%20Information.md
+  1: 'tcfeuv1',
+  2: 'tcfeuv2',
+  5: 'tcfca',
```

Comment:
okay so this is crazy

contrast 
https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/87935c4b574e974ff20ff5c1b9641374878031e2/Sections/Section%20Information.md?plain=1#L46

with 

https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/87935c4b574e974ff20ff5c1b9641374878031e2/Sections/Canada/GPPExtension%3A%20IAB%20Canada%20TCF.md?plain=1#L21

with 

https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/87935c4b574e974ff20ff5c1b9641374878031e2/Sections/Canada/GPPExtension%3A%20IAB%20Canada%20TCF.md?plain=1#L20

with 

https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/87935c4b574e974ff20ff5c1b9641374878031e2/Core/CMP%20API%20Specification.md?plain=1#L77

with 

https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/87935c4b574e974ff20ff5c1b9641374878031e2/Core/CMP%20API%20Specification.md?plain=1#L877

with

https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/87935c4b574e974ff20ff5c1b9641374878031e2/Core/CMP%20API%20Specification.md?plain=1#L1013